jobs/candlepinPerformanceTestMaintenance.groovy

```diff
@@ -0,0 +1,38 @@
+import jobLib.rhsmLib
+
+String baseFolder = rhsmLib.candlepinJobFolder
+
+job("$baseFolder/CandlepinPerformanceMaintanance") {
+    disabled()
```

Comment:
Either remove this line if we want to enable the job *or* if we don't want to enable it, just remove the cron trigger, so it needs to be manually run.

Reply:
Removed cron trigger.

Reply:
I guess we should go ahead and remove the disabled line as well, otherwise we'll have to manually enable after every job DSL run, which is annoying. It's a tiny one-line change, so I'll go ahead and do that in an update to this branch.